docker-compose.yml

```diff
@@ -4,6 +4,32 @@ services:
     container_name: fastapi
     ports:
       - "8000:8000"
+    environment:
+      EXTERNAL_API_1: http://mockoon_api_1:3001/api/data1
+      EXTERNAL_API_2: http://mockoon_api_2:3002/api/data2
+    depends_on:
+      - mockoon_api_1
+      - mockoon_api_2
+
+  mockoon_api_1:
+    image: mockoon/cli
+    container_name: mockoon_api_1
+    volumes:
+      - ./mockoon-data-1.json:/data/mockoon-data.json
+    entrypoint: ["mockoon-cli"]
+    command: ["start", "--data", "/data/mockoon-data.json"]
```

Comment:
Putting the `mockoon-cli` inline with `command` does not work in this case.
```
command: ["mockoon-cli", "start", "--data", "/data/mockoon-data.json"]
```
Results in a "mockoon-cli start is not a known command"

Thus specify the `entrypoint` explicitly.